pkg/envoy/cds/cluster.go

```diff
@@ -314,13 +345,7 @@ func localClustersFromClusterConfigs(configs []*trafficpolicy.MeshClusterConfig)
 }
 
 func getDefaultHTTPProtocolOptions() *extensions_upstream_http.HttpProtocolOptions {
```

Comment:
nit: can you replace calls to `getDefaultHTTPProtocolOptions`, with `getHTTPProtocolOptions("")` (for the purpose of readability)

Reply:
Sure, done.